docs/AUDIT.md

```diff
@@ -0,0 +1,17 @@
+# Audit
+
+Audits are conducted to ensure the absence of severe or exploitable bugs. Merge Requests are generally merged into the `master` branch without audit. The `audited` tag is used to track the latest audited commit of the `master` branch. This means that audits need to happen in order of being merged.  
```

Comment:
if audits need to happen in the order of being merged to master, does the priority even matter? whichever one is merged first has the highest priority and blocks all new stable releases if I understand correctly.

Reply:
Yes. Prioritizing would become "merging sooner".